src/endpoints/account-members.js

```diff
@@ -5,6 +5,17 @@ class AccountMembersEndpoint extends BaseExtend {
   constructor(endpoint) {
     super(endpoint)
     this.endpoint = 'account-members'
+
+    this.sendToken = (tokenRequestBody, headers = {}) =>
+    this.request.send(
```

Comment:
I don't think we need this helper method. You can just call `.send()` in all methods bellow.